src/main/java/uk/gov/hmcts/juror/api/moj/repository/jurorresponse/JurorDigitalResponseRepositoryModImpl.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.juror.api.moj.repository.jurorresponse;
+
+import com.querydsl.core.Tuple;
+import com.querydsl.core.types.dsl.CaseBuilder;
+import com.querydsl.jpa.impl.JPAQuery;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import uk.gov.hmcts.juror.api.juror.domain.ProcessingStatus;
+import uk.gov.hmcts.juror.api.moj.domain.QUser;
+import uk.gov.hmcts.juror.api.moj.domain.jurorresponse.QDigitalResponse;
+import uk.gov.hmcts.juror.api.moj.utils.SecurityUtil;
+
+import java.util.List;
+
+public class JurorDigitalResponseRepositoryModImpl implements IJurorDigitalResponseRepositoryMod {
+    @PersistenceContext
+    EntityManager entityManager;
+    private final QDigitalResponse digitalResponse = QDigitalResponse.digitalResponse;
+    private final QUser user = QUser.user;
+
+    @Override
+    public Tuple getAssignRepliesStatistics() {
+        JPAQuery<Tuple> query = getJpaQueryFactory().select(
+                new CaseBuilder().when(digitalResponse.staff.isNull()
```

Comment:
I would assume we would also want to include paper replies here?

Reply:
Checked this with liam earlier and this is for digital only